stdlib/ssl.pyi

```diff
@@ -366,6 +366,9 @@ class SSLSocket(socket.socket):
     def recvmsg(self, *args: Never, **kwargs: Never) -> Never: ...  # type: ignore[override]
     def recvmsg_into(self, *args: Never, **kwargs: Never) -> Never: ...  # type: ignore[override]
     def sendmsg(self, *args: Never, **kwargs: Never) -> Never: ...  # type: ignore[override]
+    if sys.version_info >= (3, 13):
+        def get_verified_chain(self) -> Sequence[bytes]: ...
+        def get_unverified_chain(self) -> Sequence[bytes]: ...
```

Comment:
The docstrings say that these return lists, and we generally prefer to be as precise as possible in our return types. Any reason not to do this?

```suggestion
        def get_verified_chain(self) -> list[bytes]: ...
        def get_unverified_chain(self) -> list[bytes]: ...
```

Reply:
Just wasn't sure if you prefer returning immutable types in the typeshed.